tbx/project_styleguide/templates/patterns/pages/errors/500.html

```diff
@@ -7,25 +7,52 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <style>
             html {
-                background-color: #eee;
-                font-family: sans-serif;
-                color: rgba(0, 0, 0, 0.8);
+                background-color: #1d2325;
```

Comment:
I guess our 500 page will always be dark regardless of the mode - which is probably ok. Or we could override it here when we come to do light mode.

Reply:
Yeah, I figured it wouldn't be too difficult for whoever's implementing light mode to tweak the colours.